### DISCUSSION TITLE
Miss function in some source code

### DISCUSSION THREAD

**Discussion by zonghsh888** [treeverse/lakeFS#5180] [Q&A] [answered]
I downloaded the source code of version 0.68.0 and imported it into goland. I found some functions are missing. E.g. I can't find the definition of function "HandlerFromMuxWithBaseURL" which is invoked at line 94 in pkg/api/serve.go. Can anyone help me?

**Comment by ozkatz** [comment, marked as answer]
Please take a look at the [contribution guide](https://docs.lakefs.io/contributing): specifically, you'd need to make sure you run `make build` in order to generate some of the generated code that lakeFS depends on. Hope this helps!
